WORKSPACE

```diff
@@ -67,11 +67,11 @@ http_archive(
     urls = ["https://github.com/rules-proto-grpc/rules_proto_grpc/archive/4.0.1.tar.gz"],
 )
 
-yarn_install(
+node_repositories(
+    node_version = "16.13",
     # Name this npm so that Bazel Label references look like @npm//package
```

Comment:
it's called node_version now. 

https://bazelbuild.github.io/rules_nodejs/install.html#installation-with-local-vendored-versions-of-nodejs-and-yarn

Reply:
This is actually called `nodejs` but fills the `ui/node_modules` directory which is aliased to `@npm` above. This comment should either be removed (because the name got removed) or a less misleading comment should replace it